bundle/compliance/cis_gcp/rules/cis_4_4/rule.rego

```diff
@@ -0,0 +1,17 @@
+package compliance.cis_gcp.rules.cis_4_4
+
+import data.compliance.lib.common
+import data.compliance.policy.gcp.compute.assess_instance_metadata as audit
+import data.compliance.policy.gcp.data_adapter
+
+# Ensure Oslogin Is Enabled for a Project
+finding = result {
+	# filter
+	data_adapter.is_compute_instance
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(audit.is_instance_metadata_valid("enable-oslogin", "true")),
```

Comment:
what about the exception?
```
**Exception:**
VMs created by GKE should be excluded.
VMs have names that start with `gke-` and are labeled `goog-gke-node`
```

Reply:
what about the third point?
```
Because instances can override project settings, ensure that no instance has custom metadata with key `enable-oslogin` and value `FALSE`.
```

Reply:
The console audit is being done by overviewing the compute instance [metadata page](https://console.cloud.google.com/compute/metadata?project=elastic-security-dev), which is a page that holds all the metadata configuration that is applied to the project instances.
While we're querying data about the instances themselves and verifying that the attribute is `true`.

Reply:
> what about the exception?
> 
> ```
> **Exception:**
> VMs created by GKE should be excluded.
> VMs have names that start with `gke-` and are labeled `goog-gke-node`
> ```

good catch!